docs/dev/table/sql.md

```diff
@@ -284,7 +284,7 @@ FROM Orders
     <tr>
       <td>
         <strong>Distinct</strong><br>
-        <span class="label label-primary">Batch</span>
+        <span class="label label-primary">Batch</span> <span class="label label-primary">Streaming</span>
```

Comment:
Please add the `Result Updating` label:
```
<span class="label label-info">Result Updating</span>
```

If the input table of a Distinct operation produces updates, Distinct might also produce updates, i.e., deletes.